### DISCUSSION TITLE
reComputer J1010 | J101 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#546] [Q&A]
# reComputer J1010 | J101 | Seeed Studio Wiki

reComputer J1010 | J101

https://wiki.seeedstudio.com/reComputer_J1010_J101_Flash_Jetpack/

**Comment by PHCLenzi** [comment]
Forgive my ignorance, but following the instructions in "Flashing JetPack OS via Command Line", I came across two scripts that are not mentioned where I can find them: 
./apply_binaries.sh and ./flash.sh. Where can I find these scripts?

**Reply by xzxcessarr** [reply]
Hello! Thanks for your feedback, After read the part of your mentioned, I found that this scripts is in the file Jetson-210_Linux_XXXXX_aarch64, like the file Jetson-210_Linux_R32.7.1_aarch64 you can easy download from the website such as here: https://developer.nvidia.com/embedded/linux-tegra-r3271, download the link of L4T Driver Package (BSP)
